reo/validators.py

```diff
@@ -1025,6 +1039,22 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
                         if filtered_values['min_allowable_kw'] or 0 > filtered_values['max_kw'] or 0:
                             self.input_data_errors.append('The CHP min_allowable_kw cannot be greater than its max_kw')
 
+                        # Cost curve
+                        if len(filtered_values['installed_cost_us_dollars_per_kw']) > 1:
```

Comment:
@Bill-Becker this new block of code looks exactly like the new block on lines 986-999. Could it be a sub-function that is called twice?

Reply:
Addressed in this [commit].(https://github.com/NREL/REopt_Lite_API/pull/112/commits/43495dadee6dfdb37a4e664f588da540af7b6a04)